core/utils/ethabi.go

```diff
@@ -24,36 +24,29 @@ const (
 )
 
 // ConcatBytes appends a bunch of byte arrays into a single byte array
-func ConcatBytes(bufs ...[]byte) ([]byte, error) {
-	buffer := bytes.NewBuffer([]byte{})
-	for _, b := range bufs {
-		_, err := buffer.Write(b)
-		if err != nil {
-			return nil, err
-		}
-	}
-	return buffer.Bytes(), nil
+func ConcatBytes(bufs ...[]byte) []byte {
```

Comment:
Let's get rid of this.

Reply:
Isn't `ConcatBytes` still nice to have, since it avoids the need for the empty join argument, and avoids the need to stuff the other arguments into a vector?